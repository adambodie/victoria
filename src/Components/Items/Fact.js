import React from 'react'

const Fact = ({name}) => (
	<div className='photo slide'>
		<img src={'../assets/fact.jpg'} alt='Did You Know?'/>
		<div className='photo-overlay'>
			<h3>{name}</h3> 
		</div>
	</div>
)

export default Fact
