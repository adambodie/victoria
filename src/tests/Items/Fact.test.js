import React from 'react';
import renderer from 'react-test-renderer';
import Fact from '../../Components/Items/Fact';

it('renders a fact', () => {
  const tree = renderer.create(<Fact name="Hall of Fame Football player John Elway was born in Port Angeles"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

