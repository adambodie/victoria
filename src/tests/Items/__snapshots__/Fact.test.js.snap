// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a fact 1`] = `
<div
  className="photo slide"
>
  <img
    alt="Did You Know?"
    src="../assets/fact.jpg"
  />
  <div
    className="photo-overlay"
  >
    <h3>
      Hall of Fame Football player John Elway was born in Port Angeles
    </h3>
  </div>
</div>
`;
